cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/ResourceDistributionGoal.java

```diff
@@ -257,13 +257,13 @@ protected void updateGoalState(ClusterModel clusterModel, Set<String> excludedTo
       }
     }
     if (!brokerIdsAboveBalanceUpperLimit.isEmpty()) {
-      LOG.warn("Utilization for broker ids:{} {} above the balance limit for:{} after {}.",
+      LOG.info("Utilization for broker ids:{} {} above the balance limit for:{} after {}.",
```

Comment:
Nit: Can we make this one `debug` rather than `info` -- i.e. because it specifies a soft goal violation in a usual rebalance, and we already show if the goal is `FIXED` or `VIOLATED` after the rebalance. This level of detail might be more appropriate for a debug-level.